torch/_inductor/codegen/triton.py

```diff
@@ -3084,10 +3145,14 @@ def codegen_nodes(self, nodes):
         _, (numel, rnumel) = max(nodes, key=lambda x: int(x.is_reduction())).group
 
         node_schedule = self.generate_node_schedule(nodes, numel, rnumel)
+        buf_accesses = collections.defaultdict(list)
+        for node in nodes:
+            for access in node.read_writes.reads | node.read_writes.writes:
```

Comment:
Is this ever going to be different than `get_read_names` ?

Reply:
```get_read_names``` comes with IRNode, but the node here is of ```SchedulerNode```. I think the missing type annotation caused the confusion. Let me make a PR to add more type annotations.